contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/spi/jobs.pxu

```diff
@@ -0,0 +1,43 @@
+id: ce-oem-spi/detect
+category_id: spi
+estimated_duration: 2s
+plugin: shell
+flags: also-after-suspend
+imports:
+    from com.canonical.plainbox import manifest
+requires:
+    manifest.has_spi == 'True'
+_summary:
+    To detect if the spi device exist
+_purpose:
+    Check if the SPI devices exist
+environ:
+    SPI_DEVICE_COUNT
+command:
+    spidev_test.py -d "${SPI_DEVICE_COUNT:-1}"
+
+unit: template
+template-resource: ce-oem-spi-list
+template-unit: job
+template-id: ce-oem-spi/spi-loopback-device
+_template-_summary: Test the SPI bus functionality on devices
```

Comment:
```suggestion
_template-summary: Test the SPI bus functionality on devices
```

Reply:
Commited. It's weird that `manage.py validate` did not find this error.